Makefile

```diff
@@ -148,7 +148,7 @@ export KUBECONFIG=$(KUBECONFIG_PATH)
 KUSTOMIZE_VERSION ?= v5.0.0
 CONTROLLER_TOOLS_VERSION ?= v0.14.0
 KUBECTL_VERSION ?= v1.27.0
-GINKGO_VERSION ?= v2.23.0
+GINKGO_VERSION ?= v2.23.3
```

Comment:
ideally, we should fetch the version of ginkgo from go.mod. We do something like that in metallb: https://github.com/metallb/metallb/blob/main/tasks.py#L1575